firmware/baseband/CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ set(AOPT)
 set(TOPT "-mthumb -DTHUMB")
 
 # Define C warning options here
-set(CWARN "-Wall -Wextra -Wstrict-prototypes")
```

Comment:
Was it necessary ?

Reply:
Else it's a fantastic job, and mainly new files.
I'm merging si we can all test it as quick as possible, you rock !

Reply:
Yes, I'm usinga lot of the hackrf original code and it would compile with a lot of warnings. Since I cant change the hack rf code, this is my solution.